loopdk.make

```diff
@@ -172,6 +172,11 @@ projects[saml_sp][subdir] = "contrib"
 projects[saml_sp][download][type] = "git"
 projects[saml_sp][download][url] = "http://git.drupal.org/project/saml_sp.git"
 projects[saml_sp][download][branch] = "7.x-2.x"
+; Important, so we can check if a user is signed in via SAML SP:
+projects[saml_sp][patch][] = "https://www.drupal.org/files/issues/2649458-saml_sp-is_user_authenticated_via_saml-2.patch"
+; SAML SP bugfixes:
+projects[saml_sp][patch][] = "https://www.drupal.org/files/issues/nameid-correction.patch"
```

Comment:
Are these (last two) patches needed to check if a user is logged in via saml, or are they more general fixes?